WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -404,6 +404,14 @@ private extension ProductsViewController {
     func registerTableViewCells() {
         tableView.register(ProductsTabProductTableViewCell.self)
     }
+
+    func showOrHideToolBar(resultController: ResultsController<StorageProduct>) {
```

Comment:
A couple of comments here!

-  Since `ResultsController` is part of `self`, I don't think we need to pass it as a parameter, or am I missing something?

- We try to maintain all of our methods documented, could you add some comments here, please?

Reply:
- First comment: I wanted to do a pure function, but in the end is better like you said. I'm going to use isEmpty properly.
- Second comment: No problem, comment added